### DISCUSSION TITLE
Group lists of labels into folders

### DISCUSSION THREAD

**Discussion by AlexSolovev** [organicmaps/organicmaps#741] [Ideas]
Need an ability to group several lists of labels into 1 folder, so that there won't be just 1 list of labels but 1 folder with several lists.

**Comment by rtsisyk** [comment]
I'm not sure that this UX is easy for users. Let's get some feedback from others.

**Reply by AlexSolovev** [reply]
But need to resolve somehow the issue, when there are a lot of lists of labels, that currently could be recognized only by name. At least need to add some icons / logos of specific list, filter lists by its name etc. Any other suggestions?
